src/data_validation/validation.py

```diff
@@ -0,0 +1,193 @@
+import os
+import toml
+import postcodes_uk
+import pandas as pd
+
+from src.utils.wrappers import logger_creator, time_logger_wrap, exception_wrap
+from src.utils.helpers import Config_settings
+
+# Get the config
+conf_obj = Config_settings()
+config = conf_obj.config_dict
+global_config = config["global"]
+
+# Set up logging
+validationlogger = logger_creator(global_config)
```

Comment:
You shouldn't be creating a new instance of the logger module with `logger_creator` since it gets done at the start of the pipeline. This would then reset it once again. Use this method instead.

import logging

validationlogger= logging.getLogger(__name__)

Reply:
https://jira.ons.gov.uk/browse/RDRP-244